src/features/bookOffer/components/BookDateChoice.tsx

```diff
@@ -1,14 +1,40 @@
+/* eslint-disable react-native/no-raw-text */
 import { t } from '@lingui/macro'
 import React from 'react'
 
+import { OfferStockResponse } from 'api/gen'
 import { _ } from 'libs/i18n'
 import { Typo } from 'ui/theme'
 
-export const BookDateChoice: React.FC = () => {
+interface Props {
+  stocks: OfferStockResponse[]
+}
+export const BookDateChoice: React.FC<Props> = ({ stocks }) => {
   return (
     <React.Fragment>
       <Typo.Title4>{_(t`Date`)}</Typo.Title4>
       <Typo.ButtonText>{_(t`Samedi 12 mai 2021`)}</Typo.ButtonText>
+      <Typo.Body>{`Dates réservables : ${getDatesByStatus(stocks).bookableDates}`}</Typo.Body>
+      <Typo.Body>{`Dates non-réservables : ${
+        getDatesByStatus(stocks).notBookableDates
+      }`}</Typo.Body>
     </React.Fragment>
   )
 }
+
+export const getDatesByStatus = (stocks: OfferStockResponse[]) => {
+  const bookableDates: Date[] = []
+  const notBookableDates: Date[] = []
+
+  stocks.map((stock) => {
```

Comment:
```suggestion
  stocks.forEach((stock) => {
```

si tu veux faire un side effect. map c'est pour transformer un tableau